docker/scripts/deprovision.sh

```diff
@@ -207,28 +207,30 @@ baremetal_2a2 | baremetal_2a4 | baremetal_2a5 | baremetal_hua)
 esac
 
 # Run eclypsium
-if [[ $arch == x86_64 ]]; then
-	case "$facility" in
-	ssg-* | att-*)
-		echo "skipping eclypsium in unsupported facility"
-		;;
-	*)
-		case "$class" in
-		disabled.plan.here)
-			echo "skipping eclypsium on unsuppported plan"
+if [ -z "$ECLYPSIUM_TOKEN" ] && [ "${ECLYPSIUM_TOKEN}" != "" ]; then
+	if [[ $arch == x86_64 ]]; then
```

Comment:
This test condition can be folded into the `case "$class" in` statement below. Instead of `*` you can do `*.x86.*)`. Would be nice to remove logic nesting/indentation.

Reply:
There are plans without x86 in them still, so I wouldn't combine that with the plan check for that reason.

Reply:
sigh :(. Do we have a ticket to convert over?

Reply:
wait plans maybe but class?

Reply:
Class?  plan versions?